### DISCUSSION TITLE
Getting 'Couldn't find ffmpeg or avconv' message despite having installed both?

### DISCUSSION THREAD

**Discussion by aflockdescends** [3b1b/manim#1474] [Q&A] [answered]
Hello, I hope this is the right place to post. I'm trying to get started with Manim and am having trouble with the set up. I am running Python 3.9 on macOS Catalina (10.15.7)

- I installed ffmpeg via "brew install ffmpeg" in Terminal, and also installed libav with "brew install libav". When I put "ffmpeg" or "avconv" in Terminal, all appears to be fine. 
- I installed manim via "pip3 install manim". Again, seems to be fine upon check. I also did manimgl.

I managed to locate ffmpeg, it is in /usr/local/Cellar. I don't know whether that is where it's meant to be or not. Anyway, I opened up Python using IDLE, and pasted the following 'test' code:

> from manim import *
> 
> class SquareToCircle(Scene):
>     def construct(self):
>         circle = Circle()         
>         circle.set_fill(PINK, opacity=0.5)  
>         self.play(Create(circle))

I then get the following error message:

> Warning (from warnings module):
>   File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/pydub/utils.py", line 170
>     warn("Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)
> RuntimeWarning: Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work

I'm not terribly experienced so I'm having trouble diagnosing the problem and would really appreciate some help! :)

**Comment by TonyCrane** [comment, marked as answer]
Looks like you are using the community version.
So please ask this question on discord or open an issue in [ManimCommunity/manim](https://github.com/ManimCommunity/manim)
